src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetBuilding.java

```diff
@@ -115,6 +116,16 @@ public static ToolMode getToolMode(ItemStack stack) {
         return mode;
     }
 
+    public static boolean shouldPlaceAtop(ItemStack stack) {
+        return !NBTHelper.getOrNewTag(stack).getBoolean("start_inside");
```

Comment:
Could you convert this to a constant please?

Reply:
I think that should be done in a dedicated push or PR. The code base is full of these, and there should be a coordinated effort to handle them consistently, at some point.

Reply:
ok